readme.md

```diff
@@ -234,31 +291,44 @@ source .venv/bin/activate
   To exit the venv you can run `deactivate` or close the terminal window.
 
 4. Install the package and it's dependencies
+
 ```bash
-python setup.py install
+python3 setup.py install
```

Comment:
good catch, i threw this in on a Mac that wouldn't play ball and forgot about it

Reply:
oh if it helps anywhere we can leave it, was more of a fun factoid

Reply:
ehh. probably better as it was. the issue was occurring on an ancient Mac. I've since gotten even this doorstop to run Python 3.8 (and the venv is working correctly) so screw it, let's be Pythonic

Reply:
I'm happy either way.